docs/generate_extension_categories.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+#!/usr/bin/env bash
```

Comment:
No need for two of these, just line 2 is fine.

Reply:
done